locust/test/test_zmqrpc.py

```diff
@@ -17,12 +17,12 @@ def tearDown(self):
     def test_client_send(self):
         self.client.send(Message('test', 'message', 'identity'))
         addr, msg = self.server.recv_from_client()
-        self.assertEqual(addr, b'identity')
+        self.assertEqual(addr, 'identity')
         self.assertEqual(msg.type, 'test')
         self.assertEqual(msg.data, 'message')
 
     def test_client_recv(self):
-        sleep(0.01)
+        sleep(0.1)
```

Comment:
Was the sleep not long enough?

Reply:
no, not long enough at my side, besides there's no harm to set a longer time here.